### PR TITLE
add German "Teams Translation"

### PR DESCRIPTION
**Pull request by paul-hph**
## What does this PR do?

Fixes # (issue)

notlocalized1
notlocalized

**Environment**: Production

## Type of change

Localization

## How should this be tested?

Let a German review.

### CODE REVIEW

**Review by pumfleet — APPROVED** [review]

File: apps/web/public/static/locales/de/common.json [hunk 1]
```diff
@@ -1166,5 +1166,7 @@
   "theme_dark": "Dunkel",
   "back_to_signin": "Zurück zur Anmeldung",
   "password_updated": "Passwort aktualisiert!",
-  "pending_payment": "Zahlung ausstehend"
+  "pending_payment": "Zahlung ausstehend",
+  "add_a_team": "Team hinzufügen",
+  "add_a_team": "Team löschen",
```
Comment (CarinaWolli) [inline apps/web/public/static/locales/de/common.json:1171]: @paul-siteway I think this should be "delete_team"

**Review by CarinaWolli — CHANGES_REQUESTED** [review]

File: apps/web/public/static/locales/de/common.json (hunk 1 above)
Comment (paul-siteway) [reply apps/web/public/static/locales/de/common.json]: yes! 🙈

**Review by CarinaWolli — APPROVED** [review]

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
we should not commit yarn.lock @paul-siteway can you remove the changes?

**Comment by emrysal** [comment]
@paul-siteway The lockfile shouldn't be committed at all, just left as is :)

```bash
git checkout origin/main -- yarn.lock
git commit -m "Undo yarn.lock changes"
```

**Pull request closed without merging**